### PR TITLE
Minor formatting, diagnostic and documentation changes

### PR DESCRIPTION
**Pull request by Avamander**
This PR contains very minor formatting, diagnostic and documentation changes

### CODE REVIEW

File: src/displayapp/screens/InfiniPaint.cpp [hunk 1]
```diff
@@ -7,9 +7,9 @@ using namespace Pinetime::Applications::Screens;
 extern lv_font_t jetbrains_mono_extrabold_compressed;
 extern lv_font_t jetbrains_mono_bold_20;
 
-InfiniPaint::InfiniPaint(Pinetime::Applications::DisplayApp *app, Pinetime::Components::LittleVgl& lvgl) : Screen(app), lvgl{lvgl} {
```
Comment (JF002) [inline src/displayapp/screens/InfiniPaint.cpp:10]: Not a big deal, but I prefer `Pinetime::Components::LittleVgl& lvgl` than `Pinetime::Components::LittleVgl &lvgl`.

File: src/displayapp/screens/Screen.h [hunk 2]
```diff
@@ -12,13 +12,25 @@ namespace Pinetime {
...
+           * called if the user does not touch the screen.
+           *
+           * That's why Refresh() is there: update the display periodically.
+           *
+           * @return false if the app can be closed, true if it must continue to run
+           **/
           virtual bool Refresh() = 0;
 
-          // Return false if the button hasn't been handled by the app, true if it has been handled
+          /** @return false if the button hasn't been handled by the app, true if it has been handled */
```
Comment (JF002) [inline src/displayapp/screens/Screen.h:30]: Is this notation needed for a specific documentation processor?

Comment (Avamander) [reply src/displayapp/screens/Screen.h:30]: It's the Doxygen/Javadoc format that things like CLion can parse. If you press Ctrl+Q while the cursor is on a documented variable, it'll show that documentation.

File: src/displayapp/screens/InfiniPaint.cpp (hunk 1 above)
Comment (Avamander) [reply src/displayapp/screens/InfiniPaint.cpp:10]: Do you prefer ` *` or `* `  or ` * `?

Comment (Avamander) [reply src/displayapp/screens/InfiniPaint.cpp:10]: I updated the rules to fix it.

**Comment by Avamander** [comment]
Rebased the branch, improved the code style config.

File: src/displayapp/screens/Screen.h (hunk 2 above)
Comment (JF002) [reply src/displayapp/screens/Screen.h:30]: That's really useful, thanks!

**Pull request merged**